projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/salesforce-card.js

```diff
@@ -0,0 +1,93 @@
+import { TextControl, BaseControl, ExternalLink } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import SalesforceIcon from '../../../../icons/salesforce';
+import HelpMessage from '../help-message';
+import IntegrationCard from './integration-card';
+
+const SalesforceCard = ( {
+	isExpanded,
+	onToggle,
+	data = {},
+	refreshStatus,
+	salesforceData = {},
+	setAttributes,
+} ) => {
+	const [ organizationIdError, setOrganizationIdError ] = useState( false );
+	const isValidOrgId = !! (
+		salesforceData.organizationId &&
+		salesforceData.organizationId.trim().match( /^[a-zA-Z0-9]{15,18}$/ )
```

Comment:
This is how the original code for salesforce validates the organization ID. There is a bit of duplication here, then on line 42, and active-integrations/index.js on line 36. I should centralize the logic for validating the ID. Note that we don't ever check this is a real id at salesforce - it's a simple validation check.

Reply:
Add a single method for this.